### ISSUE TITLE
browserify config required

### ISSUE THREAD

**Issue by adbl** [seatgeek/react-infinite#26]
In order to get `require('react-infinite/src/react-infinite.jsx');` working, we had to add this to `react-infinite/package.json`: 

```
...
  "browserify": {
    "transform": [ ["reactify", {"es6": true}] ]
  },
...
```

That is because this code is using JSX and es6 features, here is some more info: https://github.com/andreypopp/reactify/issues/20

Will you accept a PR with this change?

Another approach would be if you generate the transpiled `.js`-files and add them to the node module, see e.g. https://github.com/react-bootstrap/react-bootstrap/ (vanilla `.js` files end up in `react-bootstrap/lib` which is pointed out by `main` in `package.json`).

**Comment by garetht** [comment]
@adbl Yup I was planning to add that, but I imagine there's also the need to tell Browserify that the appropriate entry point is `src/react-infinite.jsx`, not the current compiled file. That's probably buried in some configuration somewhere. I'll look into that soon, but I'm happy to accept a PR for this.

**Comment by adbl** [comment]
Well, the code above was all we needed to make it work as a node_module in our tree. But then we do `require('react-infinite/src/react-infinite.jsx');`...

Not sure why you have `"main": "dist/react-infinite.js"` in `package.json`, it can point to `src/react-infinite.jsx` instead and when we can just `require('react-infinite');`. In a npm context, is this package useful _without_ browserify?

**Comment by garetht** [comment]
I point it to `dist` so that people can use this package without already having a build system for JSX and ES6 in place. Since a Node server can [render React components to strings](https://facebook.github.io/react/docs/top-level-api.html#react.rendertostring) I believe there is a use for the compiled version without Browserify.

**Comment by adbl** [comment]
True, I believe the react-router way is probably cleanest. Adding the transpiled tree (under e.g. `js/`) to your node_module and point have `"main": "js/react-infinite.js"`?

**Comment by garetht** [comment]
@adbl Fixed in `0.3.4`. Browserify's `main` path can be configured using the `browser` field.

**Comment by adbl** [comment]
cool thanks!
